server/src/main/java/org/apache/iotdb/db/engine/storagegroup/TsFileProcessor.java

```diff
@@ -77,6 +77,7 @@
 
   private TsFileResource tsFileResource;
 
+  // time range index to indicate this processor belongs to which storage group
```

Comment:
to which storage group -> to which time range

Reply:
fixed~